libsolidity/interface/CompilerStack.cpp

```diff
@@ -1020,6 +1021,10 @@ Json::Value const& CompilerStack::contractABI(Contract const& _contract) const
 		solThrow(CompilerError, "Analysis was not successful.");
 
 	solAssert(_contract.contract, "");
+	solAssert(
+		!isAnySourceExperimentalSolidity(),
+		"Experimental solidity does not support abi yet."
```

Comment:
```suggestion
		"Experimental Solidity does not support abi yet."
```

Put some respeck on our language. Same goes for the other asserts :D